common.ts

```diff
@@ -57,6 +65,10 @@ export class Config {
     return process.env.SEQUENCER_PRIVATE_KEY
   }
 
+  public static DeployerUrl(): string {
+    return process.env.DEPLOYER_URL || 'http://localhost:8080'
+  }
+
   public static DeployerPrivateKey(): string {
     return process.env.DEPLOYER_PRIVATE_KEY
```

Comment:
Ok cool so we don't need any more env vars because we used these right?

Reply:
yep